gradle/libs.versions.toml

```diff
@@ -0,0 +1,98 @@
+[versions]
```

Comment:
I'm not loving the apparent state of IDE support in this file. Not surprising since it's a new feature, but it'd be nice to have autocomplete and refactoring support for the constants. How does the developer experience compare if you define the catalog in settings.gradle?

Reply:
Yeah I think they've initial support for toml but a full extended refactoring support might be coming in future.

It has auto-complete though. Not sure if you tried or not. Refactoring support is the name itself is not yet available.

I didn't try refactoring via settings.gradle so I'm not sure about that experience.